tests/manager/core/classifiers/test_classifier.py

```diff
@@ -1187,10 +1183,8 @@ def test_staff_target_age_overrides_others(
         staff_source = DataSource.lookup(session, DataSource.LIBRARY_STAFF)
         subject1 = data.transaction.subject(type="type1", identifier="subject1")
         subject1.target_age = NumericRange(6, 8, "[)")
-        subject1.weight_as_indicator_of_target_age = 1
         subject2 = data.transaction.subject(type="type2", identifier="subject2")
         subject2.target_age = NumericRange(6, 8, "[)")
-        subject2.weight_as_indicator_of_target_age = 1
```

Comment:
`weight_as_indicator_of_target_age` isn't a property that is defined, and wasn't necessary to set for the test, so I just removed these properties.